protocol-units/bridge/README.md

```diff
@@ -0,0 +1,120 @@
+    "transaction_send_retries": 10
+  },
+  "movement": {
+    "movement_signer_address": "<MOVEMENT_PRIVATE_KEY>",
+    "movement_native_address": "0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c",
+    "mvt_rpc_connection_protocol": "http",
+    "mvt_rpc_connection_hostname": "0.0.0.0",
+    "mvt_rpc_connection_port": 30731,
+    "mvt_faucet_connection_protocol": "http",
+    "mvt_faucet_connection_hostname": "0.0.0.0",
+    "mvt_faucet_connection_port": 30732,
+    "mvt_init_network": "custom"
+  },
+  "testing": {
+    "eth_well_known_account_private_keys": [
+     <0th EL FROM ANVIL>,
+     <1st EL FROM ANVIL>,
+      ...
+    ]
+  }
+}
+
+```
+Once the relayer is up and listening for events, you can initiate a transfer.
+
+## Bridging MOVE from Ethereum to Movement without a UI
+
+First we need to create the `preimage` or secret, for the transfer. 
+
+1. `cast keccak256 <YOUR_SECRET>`
+
+The secret can be any string. When doing this operation through the UI, a unique secret is automatically generate, if using a CLI, you should always use a unique secret for each transfer.
+
+This will generate your `HASH_LOCK` value.
+
+2. Now we call `initiateBridgeTransfer` on `AtomicBridgeInitiatorMOVE` with the cast CLI
+
+```
+cast send <CONTRACT_ADDRESS> "initiateBridgeTransfer(uint256,bytes32,bytes32)" <MOVE_AMOUNT> <RECIPIENT> <HASH_LOCK> --from <YOUR_ADDRESS> --rpc-url <RPC_URL>
+```
+Where 
+- <CONTRACT_ADDRESS>: The deployed contract address.
+<MOVE_AMOUNT>: The amount of MOVE tokens (in wei) to transfer.
+- <RECIPIENT>: The recipient's address, formatted as a bytes32 string (you'll need to convert it to bytes32 beforehand).
+- <HASH_LOCK>: The hash lock for the transfer, also a bytes32 value.
+- <YOUR_ADDRESS>: Your wallet address, which will be the originator of the transaction.
+- <RPC_URL>: The RPC URL of your Ethereum node or test network.
+
+For example:
+```
+cast send 0xYourContractAddress "initiateBridgeTransfer(uint256,bytes32,bytes32)" 1000000000000000000 0x000000000000000000000000000000000000move 0x1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef --from 0xYourWalletAddress --rpc-url http://127.0.0.1:8545
+```
+The relayer will now call functions on the movement atomic bridge contract. You can inspect the logs of the service to see its progress. 
+
+Copy the `BridgeTransferId` in the logs. 
+    
+3. Now we call `complete_bridge_transfer` on the movement side. 
+```
+aptos move call \
+  --function <MODULE_ADDRESS>::<MODULE_NAME>::complete_bridge_transfer \
+  --type-args "" \
+  --args <BRIDGE_TRANSFER_ID_HEX> <PRE_IMAGE_HEX>  \
+  --profile <YOUR_PROFILE>
+```
+
+For example:
+```
+aptos move call \
+  --function 0x1::BridgeModule::complete_bridge_transfer \
+  --type-args "" \
+  --args 0x1234 676d6f7665 \
+  --profile default
+```
+You'll notice that the args in the module are of type `vec<u8>`, so you have to correctly convert any string into its hex representation. 
```

Comment:
Not clear. The arg is a 0x string, not a `Vec<u8>`. I would say that to fill the args parameters, the preimage that is a 32 bytes array should be encoded in a 0x string using `xxd` for example.